### PR TITLE
PEOPLE-105 rewrite project specs

### PR DESCRIPTION
**Pull request by mczuchnowski**
**Changes:**
- Project specs are now divided into separate files for index, show, edit and new
- Project specs now take advantage of SitePrism
- Removed webmock - it's not used in the current version of the app

**Jira:**
https://netguru.atlassian.net/browse/PEOPLE-105

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -11,18 +10,16 @@
...
 
-
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'shoulda/matchers'
 
-Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir["./spec/support/sections/*.rb"].each { |f| require f }
+Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
```
Comment (netguru-hound) [inline spec/spec_helper.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -11,18 +10,16 @@
...
 require 'site_prism'
 
-
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'shoulda/matchers'
 
-Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir["./spec/support/sections/*.rb"].each { |f| require f }
```
Comment (netguru-hound) [inline spec/spec_helper.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/features/project_filters_spec.rb [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    it 'shows only matched projects when role is provided' do
+      react_select('.filter.roles', membership.role.name)
+
+      expect(projects_page).to_not have_text('zztop')
+      expect(projects_page).to have_text('test')
+    end
+  end
+
+
```
Comment (netguru-hound) [inline spec/features/project_filters_spec.rb]: Extra blank line detected.

**Comment by blase** [comment]
I still remember `:project_zztop` and Developer Daisy. Oh, the memories...

:+1:

**Pull request merged**